rdr_service/etl/etl.sql

```diff
@@ -529,6 +529,37 @@ CREATE TABLE cdm.dose_era
     UNIQUE KEY (id)
 );
 
+-- -----------------------------------------------
+-- questionnaire_vibrent_forms
+-- -----------------------------------------------
+DROP TABLE IF EXISTS cdm.questionnaire_vibrent_forms;
+
+CREATE TABLE cdm.questionnaire_vibrent_forms (
+    questionnaire_id            bigint,
+    version                     bigint,
+    vibrent_form_id             varchar(1024)
+);
+
+INSERT INTO cdm.questionnaire_vibrent_forms
+SELECT
+    qh.questionnaire_id,
+    qh.version,
+    json_unquote(json_extract(convert(qh.resource using utf8), '$.identifier[0].value')) vibrent_form_id
+FROM questionnaire_history qh
+;
+
```

Comment:
You should create a compound index on cdm.questionnaire_vibrent_forms with the 'questionnaire_id' and 'version' columns here so joinning is fast.

Reply:
Good idea, done!